src/static/js/modules/jquery/cf_inputSplit.js

```diff
@@ -4,7 +4,8 @@
 
 'use strict';
 
-require( 'jquery' );
+var $ = require( 'jquery' );
+var jQuery = $;
```

Comment:
No need to define it here, we should strip it from the self-executing function (especially since `$` gets redefined there).